src/ui/components/Config/ConfigRow.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import axios from 'axios';
+import { useQueryClient } from 'react-query';
+import TextField from '@material-ui/core/TextField';
+import Switch from '@material-ui/core/Switch';
+
+import PropTypes from 'prop-types';
+import useStyles from './styles';
+
+const ConfigRow = props => {
+  const classes = useStyles();
+  const queryClient = useQueryClient();
+  const { option } = props;
+  const [value, setValue] = useState(option.value);
+  const [changed, setChanged] = useState(false);
+
+  const isBool = useMemo(() => {
+    let val = option.value;
+    val = val === 'true' ? true : val === 'false' ? false : val;
```

Comment:
Is it easier to do this than just change the physical variable type to be a boolean?

Reply:
As a side note: it seems like once I edit the value it saves to the database as the boolean type anyway. So I think editing this directly in the initial configVars should work. Just make sure the places before which were checking the `configUtil.getRequireAuth() === 'false'` and `configUtil.getRequireAuthOutgoing() === 'false'` are updated accordingly